Files/View Models/InteractionViewModel.cs

```diff
@@ -53,7 +53,10 @@ public void CheckForImage()
             {
                 string ItemExtension = (App.CurrentInstance.ContentPage as BaseLayout).SelectedItem.FileExtension;
 
-                if (ItemExtension == ".png" || ItemExtension == ".jpg" || ItemExtension == ".bmp" || ItemExtension == ".jpeg")
+                if (ItemExtension.ToLower() == ".png" 
```

Comment:
When I tried doing this the application would crash when right clicking on an image file, have you confirmed that this works?

Reply:
It is string, ToLower() can't crash app. From my side it is working fine.

Reply:
I would prefer using ItemExtension.Equals(".png", StringComparison.OrdinalIgnoreCase) for this one.

Reply:
Just to keep things consistent.

Reply:
Done

Reply:
@tsvietOK Can you test that the functionality still works after doing the changes @duke7553 requested?

Reply:
Yeah, i will test it little bit later

Reply:
@yaichenbaum It is working